spec/controllers/problems_controller_spec.rb

```diff
@@ -1,3 +1,14 @@
+# FIXME: This is a monkey patch to workaround an issue in rspec-rails:
+#
+#   NoMethodError:
+#     undefined method `cache' for nil:NilClass
+#
+# See: https://github.com/rspec/rspec-rails/issues/1532#issuecomment-174679485
+#
+RSpec::Rails::ViewRendering::EmptyTemplatePathSetDecorator.class_eval do  
```

Comment:
It may be worth guarding this on the bundled version of `rspec-rails` and raising an error if it's not as expected. This will save potential hair-pulling down the line.